quality_control/views/qc_test_view.xml

```diff
@@ -0,0 +1,143 @@
+                            <field name="fill_correct_values" />
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                            />
+                        </group>
+                    </group>
+                    <field name="test_lines" nolabel="1">
+                        <tree string="Questions">
+                            <field name="sequence" widget="handle" />
+                            <field name="name" />
+                            <field name="type" />
+                            <field name="min_value" />
+                            <field name="max_value" />
+                            <field name="uom_id" />
+                            <field name="ql_values" />
+                        </tree>
+                    </field>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="qc_test_tree_view">
+        <field name="name">qc.test.tree</field>
+        <field name="model">qc.test</field>
+        <field name="arch" type="xml">
+            <tree string="Tests">
+                <field name="name" />
+                <field name="category" />
+                <field name="type" />
+                <field name="object_id" />
+                <field name="company_id" groups="base.group_multi_company" />
+            </tree>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="action_qc_test">
+        <field name="name">Tests</field>
+        <field name="res_model">qc.test</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <record model="ir.ui.view" id="qc_test_question_form_view">
+        <field name="name">qc.test.question.form</field>
+        <field name="model">qc.test.question</field>
+        <field name="arch" type="xml">
+            <form string="Test question">
+                <label for="name" class="oe_edit_only" />
+                <h1>
+                    <field name="name" class="oe_inline" />
+                </h1>
+                <group>
+                    <field name="sequence" />
+                    <field name="type" />
+                </group>
+                <group
+                    name="qualitative"
+                    string="Answers"
+                    colspan="4"
+                    attrs="{'invisible': [('type', '!=', 'qualitative')]}"
+                >
+                    <field
+                        name="ql_values"
+                        nolabel="1"
+                        attrs="{'required': [('type','=','qualitative')]}"
+                    >
+                        <tree string="Question value" editable="bottom">
+                            <field name="name" />
+                            <field name="ok" />
+                        </tree>
+                    </field>
+                </group>
+                <div
```

Comment:
Maybe this div could be improved since values appear very big and its not clear what they are supposed to be.